### ISSUE TITLE
Remove '#' from URL

### ISSUE THREAD

**Issue by KVGarg** [coala/projects#717]
Requires basic knowledge of angularJS to solve this issue

A newcomer issue

**Comment by theParadoxicalWizard** [comment]
Can you please assign this issue to me?

**Comment by li-boxuan** [comment]
Why do we need to remove it?

**Comment by theParadoxicalWizard** [comment]
The # is unnecessary, wouldn't it be better if we remove it
But then we also have to create mechanism that redirects everybody from with-# to non-#

**Comment by theParadoxicalWizard** [comment]
@KVGarg @li-boxuan i want to work on it , please assign me ..

**Comment by KVGarg** [comment]
@li-boxuan To prettify the URL, it will be good to remove the #
Using AngularJS framework, the # is always appended to a URL.
AngularJS has a built-in method(one line addition), that doesn’t append # to URL.

**Comment by theParadoxicalWizard** [comment]
Is there any way to use asdf-vm on windows,
there docs talk only about linux and mac

**Comment by KVGarg** [comment]
> Is there any way to use asdf-vm on windows,
> there docs talk only about linux and mac

@theParadoxicalWizard Is this related to current issue?

**Comment by theParadoxicalWizard** [comment]
> > Is there any way to use asdf-vm on windows,
> > there docs talk only about linux and mac
> 
> @theParadoxicalWizard Is this related to current issue?

I am really sorry for that,
Can i be assigned to this issue so as to create a pull request that would solve this

**Comment by KVGarg** [comment]
I don’t have permissions to assign you. Please wait for few hrs, if it’s a valid issue you will be assigned to it.

**Comment by theParadoxicalWizard** [comment]
@li-boxuan can you clarify if this is a valid issue?

**Comment by abhishalya** [comment]
No, this is not as easy as that. The problem here is that the project urls are used widely and changing the links does no good to them. We at least need to make sure that they have a redirect to the working url.
